cdk/helpers/lambdas/checkSumOfFiles.spec.ts

```diff
@@ -1,9 +1,11 @@
 import path from 'node:path'
 import { checkSumOfFiles } from './checksumOfFiles.js'
+import assert from 'node:assert'
```

Comment:
`node:assert` is considered legacy code. The recommended option to use in order with the documentation is `node:assert/strict`. 

Check https://github.com/NordicSemiconductor/asset-tracker-cloud-coiote-aws-converter-js/pull/85/commits/b68542092fa3713da2f13c0d28c90642dbe3988b for more info.